pkg/selinux/selinux.go

```diff
@@ -46,6 +46,8 @@ const (
 	selinuxPath      = "/sys/fs/selinux"
 	xattrNameSelinux = "security.selinux"
 	stRdOnly         = 0x01
+
+	mcsdir = "/var/run/rkt/mcs" // Directory to use for MCS storage
```

Comment:
This hardcodes the MCS directory in the `selinux` package which makes it rkt-specific. Maybe we can find a way to not do that?